assets/js/lib/test-utils/data/databases.js

```diff
@@ -0,0 +1,116 @@
+export default [
```

Comment:
The databases slice default state added, as this is how it is loaded in the initial fetch (and next updates)